packages/flutter_fuchsia/lib/src/flutter_views.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2018 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:io';
+
+import 'package:process/process.dart';
+import 'package:logging/logging.dart';
+
+import 'dart/fuchsia_dart_vm.dart';
+import 'fuchsia_device_command_runner.dart';
+
+final String ipv4Loopback = InternetAddress.LOOPBACK_IP_V4.address;
+
+final Logger _log = new Logger('flutter_fuchsia::flutter_views');
+
+final ProcessManager _processManager = new LocalProcessManager();
+
+/// Persistent VM service cache to avoid repeating handshakes across function
+/// calls.
+final HashMap<int, FuchsiaDartVm> _fuchsiaDartVmCache =
+    new HashMap<int, FuchsiaDartVm>();
+
+/// Returns a list of flutter views for the given [ipv4Address].
+/// TODO(awdavies): just returns flutter view names. Needs to return
+/// FlutterView objects of some kind that contain a JSON RPC peer.
```

Comment:
To reduce future breaking changes, let's start with a `FuchsiaFlutterView` class that has a single field `String name`. In the future we can add more methods to it without breaking existing clients.

Reply:
sgtm

Reply:
Done.